index.html

```diff
@@ -0,0 +1,164 @@
+    </div>
+    <div class="container">
+        <div class="image">
+            <img class="img" src="./images/tonic_2.png" alt="">
+        </div>
+        <div class="content">
+            <h2 class="content-header">Tonic</h2>
+            <ul class="specification">
+                <li class="name">Aliza</li>
+                <li class="skills"><img src="./images/dot.png" alt=""> Back End Dev</li>
+                <li class="year"><img src="./images/dot.png" alt=""> 2015</li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+    <div class="container">
+        <div class="image">
+            <img class="img" src="./images/multi_2.png" alt="">
+        </div>
+        <div class="content">
+            <h2 class="content-header">Multi-Post <br>
+              Stories</h2>
+            <ul class="specification">
+                <li class="name">Aliza</li>
+                <li class="skills"> <img src="./images/dot.png" alt=""> Back End Dev</li>
+                <li class="year"><img src="./images/dot.png" alt="">2015</li>
+            </ul>
+            <p class="content-para">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias consectetur nobis quod tempora, ad illum!</p>
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+</div>
+<div class="about-container">
+
+  <div class="about">
+<h2 class="about-heading">
+  About Myself
+</h2>
+<p class="about-para">
+  Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h4 class="connect">LET'S CONNECT</h4>
+<ul class="about-icons">
+  <li class="twitter"><a href=""><img src="./images/twitter.png" ></a> </li>
+  <li class="linkedln"><a href=""><img src="./images/Linkedin.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/gmail.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/github.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/Vector.png" alt=""></a></li>
+
+</ul>
+<button class="get-resume">get my resume</button>
+  </div>
+<div >
+  <div class="language-section" >
+    <p class="languages">Languages </p>
```

Comment:
Kindly ensure that you use the semantic tag `<ul>`. This should be a list of lists, that is language, Skills, and Frameworks are all items of a list
inside language `<li>` and have another list with javascript, HTML, and CSS.
<img width="1673" alt="Screenshot 2023-04-29 at 08 49 20" src="https://user-images.githubusercontent.com/48546992/235286565-ab850fed-c5bd-44ab-a8b3-4426ce332dae.png">